src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -151,6 +152,22 @@ public void updateFilteredAppointmentList(Predicate<Appointment> predicate) {
         filteredAppointments.setPredicate(predicate);
     }
 
+    @Override
+    public UniqueAppointmentList getAppointments() {
+        return (UniqueAppointmentList) this.wellNus.getAppointmentList();
+    }
+
+    @Override
+    public boolean hasAppointment(Appointment appointment) {
+        requireNonNull(appointment);
+        return this.getAppointments().contains(appointment);
+    }
+
+    @Override
+    public void addAppointment(Appointment appointment) {
+        this.getAppointments().add(appointment);
+    }
```

Comment:
get rid of getApointments() function, follow the syntax of the other functions in this file and change hasAppointment() and addAppointment() to wellNus.addAppointment(appointment). Then create a method for addAppointment() in WellNus class

Reply:
Also recommend to put it above removeAppointment() method instead of at the end.